### PR TITLE
Add LGTM check

### PR DESCRIPTION
**Pull request by tyll**
This change is [Reviewable](https://reviewable.io/reviews/linux-system-roles/network/131)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@ linux-system-roles/network
 [![Coverage Status](https://coveralls.io/repos/github/linux-system-roles/network/badge.svg)](https://coveralls.io/github/linux-system-roles/network)
 [![Travis Build Status](https://travis-ci.org/linux-system-roles/network.svg?branch=master)](https://travis-ci.org/linux-system-roles/network)
 [![Code Style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
+[![Language grade: Python](https://img.shields.io/lgtm/grade/python/g/linux-system-roles/network.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/linux-system-roles/network/context:python)
```
Comment (pcahyna) [inline README.md]: I would prefer to go in the opposite direction and remove those banners. Status of those checks should be visible in the CI reports.

Comment (tyll) [reply README.md]: IMHO the banners are nice advertising for users/contributors. They would not see the status in the PRs.

**Comment by tyll** [comment]
[citest bad]

**Comment by tyll** [comment]
[citest bad]

**Pull request merged**